### PR TITLE
feat: Portable Debug Data images

### LINKED ISSUE #15: Raygun.Blazor: PEDebug data populated where appropriate
**Please describe your new feature request**

As listed in the PR #1 TODO list

**Describe the solution you'd like**

- TBD

**Describe alternatives you've considered**

- TBD

**Additional context**

- TBD

### PR DESCRIPTION
**Pull request by miquelbeltran**
### Description :memo:

- **Purpose**: Implements obtaining PE Debug data
- **Approach**: Port Raygun4Net solution to the Blazor provider
- Closes #15 

**Type of change**

- [x] New feature (non-breaking change which adds functionality)

**Updates**

- Implements providing the Portable Debug Data image references based on the Raygun4Net implementation.
- They get attached in the `error.images` payload (see README changes for example)
- Adds unit tests that verify the inclusion of stacktraces and the pdb images
- Adds info about stacktraces in the README

### Test plan :test_tube:

- Tested on WebAssembly, Server and MAUI sample apps
- Unit tests

### Author to check :eyeglasses:

- [x] Project and all contained modules builds successfully
- [x] Self-/dev-tested
- [x] Unit/UI/Automation/Integration tests provided where applicable
- [x] Code is written to standards
- [x] Appropriate documentation written (code comments, internal docs)

### Reviewer to check :heavy_check_mark:

- [ ] Project and all contained modules builds successfully
- [ ] Change has been dev-/reviewer-tested, where possible
- [ ] Unit/UI/Automation/Integration tests provided where applicable
- [ ] Code is written to standards
- [ ] Appropriate documentation written (code comments, internal docs)

### CODE REVIEW

**Comment by miquelbeltran** [comment]
I will wait until the PDB support for Blazor clients is available on raygun.com and test this end-to-end, we can keep this PR until then

File: src/Raygun.Blazor/Models/StackTraceDetails.cs [hunk 1]
```diff
@@ -87,10 +87,22 @@ internal StackTraceDetails(StackFrame frame)
             ColumnNumber = frame.GetFileColumnNumber();
             FileName = frame.GetFileName();
             ILOffset = frame.GetILOffset();
-            //ImageSignature = frame.ImageSignature;
             LineNumber = frame.GetFileLineNumber();
             MethodName = names.MethodName;
             MethodToken = frame.GetMethod()?.MetadataToken;
+
+            // Attempt to get the image signature from the debug information
```
Comment (ProRedCat) [inline src/Raygun.Blazor/Models/StackTraceDetails.cs:94]: You should test whether this works on MAUI Blazor Hybrid apps specifically in Android. We had to do quite a bit of work to get the image there. 

Check out this PR for some more context https://github.com/MindscapeHQ/raygun4maui/pull/20

Comment (miquelbeltran) [reply src/Raygun.Blazor/Models/StackTraceDetails.cs:94]: Thanks for the hint! When I did the testing on Android, I was able to obtain the PE Debug data from the frames. The only platform where it didn't work was WebAssembly Blazor apps. I will check again once when we have the solution working end to end on Blazor Server.

**Review by ProRedCat — APPROVED** [review]
Lgtm, thanks for covering the case of Android Assembly Store

**Pull request merged**